package.json

```diff
@@ -46,6 +48,7 @@
     "rollup": "^2.26.2",
     "rollup-plugin-preserve-shebang": "^1.0.1",
     "semver-compare": "^1.0.0",
+    "shx": "^0.3.3",
```

Comment:
Thoughts on using `rimraf` instead?

https://www.npmjs.com/package/rimraf

Reply:
No difference. I use shx because it includes other things that I usually need. 

https://github.com/shelljs/shelljs

Reply:
Is this holding the PR back? I can change to rimraf if you want.

Reply:
I was on a vacation earlier, got back recently

Reply:
Thank you! No problem.